### PR TITLE
Enable parallel build in Concourse.

### PR DESCRIPTION
**Pull request by hlinnaka**
Let's see if this works on the PR pipeline..

### CODE REVIEW

**Comment by hlinnaka** [comment]
Now that was a successful experiment. It compiled in 12.5 minutes in the PR pipeline, compared to about 16 minutes normally.

**Comment by hlinnaka** [comment]
Pushed a slightly cleaned up version to this PR. Please review

File: concourse/scripts/compile_gpdb.bash [hunk 1]
```diff
@@ -74,9 +74,9 @@ function make_sync_tools() {
 function build_gpdb() {
   pushd gpdb_src/gpAux
     if [ -n "$1" ]; then
-      make "$1" GPROOT=/usr/local dist
+      make "$1" GPROOT=/usr/local PARALLEL_MAKE_OPTS=-j4 dist
```
Comment (ashwinstar) [inline concourse/scripts/compile_gpdb.bash]: Why 4 can we make it based on something maybe cores count from `/proc/cpuinfo`

**Comment by hlinnaka** [comment]
On 02/10/2017 09:08 PM, Ashwin Agrawal wrote:
> ashwinstar commented on this pull request.
>
>
>
>> @@ -74,9 +74,9 @@ function make_sync_tools() {
>  function build_gpdb() {
>    pushd gpdb_src/gpAux
>      if [ -n "$1" ]; then
> -      make "$1" GPROOT=/usr/local dist
> +      make "$1" GPROOT=/usr/local PARALLEL_MAKE_OPTS=-j4 dist
>
> Why 4 can we make it based on something maybe cores count from `/proc/cpuinfo`

Well, that's more or less what the removed PARALLEL_BUILD tried to do. 
Yeah, we could've fixed that instead of just nuking it. In practice, 
however, I don't think there's much to be gained from going above 4, and 
4 is small enough to be fairly harmless even on a single-core system.

(See also the commit message, I tried to explain this there)

- Heikki

**Comment by ashwinstar** [comment]
Okay will be helpful for future to add the commit message justification as comment in code.

**Comment by ashwinstar** [comment]
Also might be worth modifying this file as well compile_gpdb_custom_config.bash

**Comment by hlinnaka** [comment]
On 02/10/2017 09:18 PM, Ashwin Agrawal wrote:
> Okay will be helpful for future to add the commit message
> justification as comment in code.

Ok, fair enough. Added a comment.

> Also might be worth modifying this file as well
> compile_gpdb_custom_config.bash

Thanks, added -j4 there as well.

Updated the patch, please have one final look. Assuming it still passes 
on the concourse instance, I will push this.

- Heikki

**Comment by ashwinstar** [comment]
LGTM Thank You.

**Comment by hlinnaka** [comment]
Pushed, thanks for the review!

**Pull request closed without merging**